app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -176,10 +177,17 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
             SyncEngine.Bookmarks -> R.string.pref_key_sync_bookmarks
             SyncEngine.Passwords -> R.string.pref_key_sync_logins
             SyncEngine.Tabs -> R.string.pref_key_sync_tabs
+            SyncEngine.CreditCards -> R.string.pref_key_sync_credit_cards
+            SyncEngine.Addresses -> R.string.pref_key_sync_address
             else -> throw IllegalStateException("Accessing internal sync engines")
         }
 
-        listOf(SyncEngine.History, SyncEngine.Bookmarks, SyncEngine.Tabs).forEach {
+        listOf(
+            SyncEngine.History,
+            SyncEngine.Bookmarks,
+            SyncEngine.Tabs,
+            SyncEngine.Addresses
```

Comment:
Are wee missing CreditCards here?

Reply:
Nope, credit cards and passwords are below (line 200 & 216)

Reply:
Yes, this makes sense now.